packages/apps/project/src/components/molecules/DropdownTaskCharacteristics/DropdownTaskCharacteristics.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {
+  LabelText,
+  TagList,
+  bootstrapColors,
+  checkNullString,
+} from '@axelor/aos-mobile-ui';
+import {useTranslator} from '@axelor/aos-mobile-core';
+
+interface DropdownTaskCharacteristicsProps {
+  style?: any;
+  projectTaskCategory?: any;
+  projectTaskSection?: any;
+  targetVersion?: any;
+  taskDate?: string;
+  taskEndDate?: string;
+  taskDeadline?: string;
+  projectTaskTagSet?: any;
+}
+
+const DropdownTaskCharacteristics = ({
+  style,
+  projectTaskCategory,
+  projectTaskSection,
+  targetVersion,
+  taskDate,
+  taskEndDate,
+  taskDeadline,
+  projectTaskTagSet,
+}: DropdownTaskCharacteristicsProps) => {
+  const I18n = useTranslator();
+
+  const tagsList = useMemo(() => {
+    return projectTaskTagSet?.map(tag => ({
+      title: tag?.name,
+      color: bootstrapColors[tag?.colorSelect],
```

Comment:
bootstrap colors are added to all theme, use UseThemeColor like always to get tag color